setup.py

```diff
@@ -13,7 +13,7 @@
     "pytest-asyncio",
     "pytimeparse",
     "anyio",
-    "chia-blockchain==2.1.4",
+    "chia-blockchain==2.2.1",
```

Comment:
I don't think we can bump the version here yet because we need chia-blockchain to update clvm_tools_rs to 0.1.41 which is coming in the next release.